### ISSUE TITLE
Estadísticas de una encuentas

### ISSUE THREAD

**Issue by claudiaguzman** [IHC-2-2015/manuel_api#9]
El alumno debe seleccionar una encuesta y dado el correo de este y el id de la encuesta se debe saber a qué grupo pertenece y si todo el grupo contesto la encuesta.
Si el grupo si completo la encuesta seleccionada se debe indicar si el alumno es jefe de grupo o proyecto o simplemente integrante normal de un grupo. Par cada uno de los casos debe suceder lo siguiente:
1. En caso de no ser jefe ni de grupo ni de proyecto se deben mostrar solo las estadísticas del alumno en particular, para esto es necesario obtener los promedios del alumno y del grupo con respecto a cada pregunta, además es necesario obtener los enunciados de las preguntas para que el alumno pueda verlos.
2. En caso de ser jefe de grupo se debe entregar el promedio del grupo con respecto a cada pregunta y el listado de compañeros. Al momento de seleccionar a un compañero se debe entregar el promedio de este con respecto a cada pregunta. Nuevamente también es necesario que se entreguen el enunciado de cada pregunta.
3. En caso de ser jefe de proyecto primero se deben entregar los datos de cada grupo del curso (id y nombre) y al momento de seleccionar un grupo debe suceder lo mismo que el caso 2.

**Comment by CamilaMarin** [comment]
Esto complementa a la MA-028 del sprint anterior

**Comment by franciscoavello** [comment]
Lo más urgente son dos funciones. Primero una función que me devuelva si es que soy jefe de grupo de tal grupo y otra función que devuelva si soy jefe de proyecto de un curso.

**Comment by CamilaMarin** [comment]
ok

**Comment by vaskiwi** [comment]
Se pide prioridad sobre devolver si soy jefe de proyecto de un curso, ya que el jefe de grupo se puede sacar desde consultas que ya están implementadas.

**Comment by CamilaMarin** [comment]
ok trabajando en eso

**Comment by claudiaguzman** [comment]
Esta consulta va a tener un leve cambio, ya que hablando con el PO nos indico que el jefe de proyecto debe comparar promedio de los grupos con el del curso.

**Comment by claudiaguzman** [comment]
Para hacer la consulta del jefe de proyecto es necesario entregar el id del curso, sin embargo, para eso es necesario saber previamente cual es el curso asociado a la encuesta seleccionada

**Comment by CamilaMarin** [comment]
Entendiendo que el alumno puede ser jefe de proyecto en más de un curso , quizás sería mejor que con `alumno_id` se entreguen todos los cursos en los que ese alumno es jefe de proyecto y luego para un curso en particular se entreguen las encuestas correspondientes, ya que como jefe de proyecto debería poder ver las estadísticas de las diferentes encuestas enviadas al curso.

**Comment by claudiaguzman** [comment]
Lo que pasa es que el alumno selecciona encuestas, por lo tanto la encuesta seleccionada deberia estar asociada a un curso. Puede ser que esto se solucione con la petición #25

**Comment by CamilaMarin** [comment]
Pero entonces ¿el jefe de proyecto selecciona la encuesta antes de seleccionar el curso ?
La parte de los alumnos si se solucionaría con la petición #25 pero no estoy segura que sea de la misma forma para el jefe de proyecto.

**Comment by claudiaguzman** [comment]
Sí, todos los alumnos seleccionan encuestas solo que las vistas cambian dependiendo si el alumno es jefe de grupo o proyecto del curso con el cual la encuesta esta asociada

**Comment by CamilaMarin** [comment]
ok
